apis/r/tests/testthat/test-SCEOutgest.R

```diff
@@ -0,0 +1,424 @@
+test_that("Load SCE object from ExperimentQuery mechanics", {
```

Comment:
Totally agree! 

My only suggestion is that we wrap all of the code used to create these test experiments into a helper function since we're performing lagely the same steps in test-SeuratOutgest-object, test-SeuratOutgest-reduction, and now test-SCEOutgest.

Or perhaps we update the existing helper, `create_and_populate_experiment()`, to optionally populate the other slots.

Reply:
`create_and_populate_experiment()` has been expanded to allow creation of `obsm`, `varm`, `obsp`, and `varp` layers; the updated functionality has been used for both Seurat and SCE tests